jobs/JGFS_ATMOS_GEMPAK_META

```diff
@@ -47,8 +46,8 @@ export COMINnam=${COMINnam:-$(compath.py "${envir}/nam/${nam_ver}")/nam}
 GRID=1p00 YMD=${PDY} HH=${cyc} declare_from_tmpl -rx "COMIN_ATMOS_GEMPAK_1p00:COM_ATMOS_GEMPAK_TMPL"
 
 GRID="meta" YMD=${PDY} HH=${cyc} declare_from_tmpl -rx "COMOUT_ATMOS_GEMPAK_META:COM_ATMOS_GEMPAK_TMPL"
-if [[ ! -d "${COMOUT_ATMOS_GEMPAK_META}" ]] ; then
-  mkdir -m 775 -p "${COMOUT_ATMOS_GEMPAK_META}"
+if [[ ! -d "${COMOUT_ATMOS_GEMPAK_META}" ]]; then
+    mkdir -m 775 -p "${COMOUT_ATMOS_GEMPAK_META}"
 fi
```

Comment:
No need to check for existence.  `-p` with `-m` will not raise an error if the directory already exists.

mkdir -m 775 -p "${COMOUT_ATMOS_GEMPAK_META}"